resources/event-bus/charts/publish-knative/templates/_helpers.tpl

```diff
@@ -1,15 +1,11 @@
 {{- define "publish-knative.name" -}}
-{{- printf "publish-knative" -}}
+{{- printf "%s-%s" .Release.Name "publish" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
 {{- define "publish-knative.fullname" -}}
```

Comment:
Do we still need the **publish-knative.fullname**?

Reply:
No, will remove it